tests/test_get_all_sentences.rs

```diff
@@ -65,3 +75,88 @@ fn test_get_all_sentences_returns_200() {
         2,
     );
 }
+
+#[test]
+fn test_get_paginated_sentences() {
+
+    let connection = db::get_connection();
+    db::clear(&connection);
+
+    let english_iso639_3 = "eng";
+    db::insert_language(
+        &connection,
+        &english_iso639_3,
+    );
+
+    let base_url = format!(
+        "{}/sentences",
+        tests_commons::SERVICE_URL,
+    );
+
+    /* insert multiple sentences with different
+       content and "consecutives" uuids,
+       from a full-zeros uuid to x-x-x-x-...13 */
```

Comment:
something is wrong if you need to make consecutive uuid (see my comment above) , i.e you're tricking the reality to please your test though you should do the oppositve (i.e in reality multiple sentences add in order A B C don't have any garantee to have uuid in order  A B C )